### ISSUE TITLE
Errors on python2.7

### ISSUE THREAD

**Issue by LightToYang** [xxradon/PytorchToCaffe#53]
```
/PytorchToCaffe/analysis/layers.py", line 174
    self.out = Blob([self.batch_size, num_out, *outs], self)
                                                ^
SyntaxError: invalid syntax
```

**Comment by LightToYang** [comment]
```
Traceback (most recent call last):
  File "example/resnet_pytorch_2_caffe.py", line 19, in <module>
    pytorch_to_caffe.trans_net(resnet18,input,name)
  File "./pytorch_to_caffe.py", line 762, in trans_net
    out = net.forward(input_var)
  File "/usr/local/lib/python2.7/dist-packages/torchvision/models/resnet.py", line 196, in forward
    x = self.conv1(x)
  File "/usr/local/lib/python2.7/dist-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/torch/nn/modules/conv.py", line 320, in forward
    self.padding, self.dilation, self.groups)
  File "./pytorch_to_caffe.py", line 658, in __call__
    for stack in traceback.walk_stack(None):
AttributeError: 'module' object has no attribute 'walk_stack'

```

**Comment by IsDora** [comment]
hello，i have the same question with you.
Have  you solved the problem with python 2.7?
